### DISCUSSION TITLE
Save sorting of Stars overview

### DISCUSSION THREAD

**Discussion by serkonda7** [community/community#16759] [Other Feature Feedback, Questions, & Ideas]
The default sorting in `Your Stars` is by date starred. However I think the recently active option provides more value so everytime I open my stars, I change the sorting. Could this be saved inside a  cookie or per account, to save a few clicks.

**Comment by Griffin641498** [comment]
> The default sorting in `Your Stars` is by date starred. However I think the recently active option provides more value so everytime I open my stars, I change the sorting. Could this be saved inside a cookie or per account, to save a few clicks.
